packages/react/src/hooks/constants.ts

```diff
@@ -0,0 +1,272 @@
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  phone_number_verified: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  picture: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  preferred_username: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  profile: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  sub: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  updated_at: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  website: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  zoneinfo: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  email: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+  phone_number: {
+    isUpdated: true,
+    nextStep: {
+      codeDeliveryDetails: {
+        attributeName: undefined,
+        deliveryMedium: undefined,
+        destination: undefined,
+      },
+      updateAttributeStep: "DONE",
+    },
+  },
+};
+// Default object to be passed into useDataState
+export const defaultSendUserAttributeVerificationCodeOutput: SendUserAttributeVerificationCodeOutput =
+  {
+    attributeName: "email",
+    deliveryMedium: "EMAIL",
+    destination: "a***@b.com",
+  };
```

Comment:
What if the configuration of the consuming project has only enabled `phone_number` as a verifiable attribute?